Investigate java.lang.ClassNotFoundException in Faces-4.0

Enabling EE10 in tests, found an exception using faces-4.0 with an xhtml with the line `application implicit object test: <h:outputText id="out1" value="#{application.servletContextName}"/> ` . When accessing **application.servletContextName** hit the following exception.

> Exception thrown by application class 'jakarta.faces.webapp.FacesServlet.service:255'
> jakarta.servlet.ServletException: com.ibm.wsspi.webcontainer.facade.ServletContextFacade
> at jakarta.faces.webapp.FacesServlet.service(FacesServlet.java:255)
> at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1258)
> at [internal classes]
> Caused by: java.lang.NoClassDefFoundError: com.ibm.wsspi.webcontainer.facade.ServletContextFacade
> at .apply(Unknown Source)
> at org.apache.myfaces.el.resolver.LambdaBeanELResolver.getValue(LambdaBeanELResolver.java:75)
> at [internal classes]
> at jakarta.faces.component._DeltaStateHelper.eval(_DeltaStateHelper.java:339)
> at jakarta.faces.component.UIOutput.getValue(UIOutput.java:69)
> at org.apache.myfaces.renderkit.RendererUtils.getValue(RendererUtils.java:331)
> at org.apache.myfaces.renderkit.RendererUtils.getStringValue(RendererUtils.java:173)
> at org.apache.myfaces.renderkit.html.base.HtmlTextRendererBase.renderOutput(HtmlTextRendererBase.java:132)
> at org.apache.myfaces.renderkit.html.base.HtmlTextRendererBase.encodeEnd(HtmlTextRendererBase.java:122)
> at jakarta.faces.component.UIComponentBase.encodeEnd(UIComponentBase.java:634)
> at jakarta.faces.component.UIComponentBase.encodeAll(UIComponentBase.java:523)
> at jakarta.faces.component.UIComponentBase.encodeAll(UIComponentBase.java:519)
> at jakarta.faces.component.UIComponentBase.encodeAll(UIComponentBase.java:519)
> at jakarta.faces.component.UIComponentBase.encodeAll(UIComponentBase.java:519)
> at org.apache.myfaces.view.facelets.FaceletViewDeclarationLanguage.renderView(FaceletViewDeclarationLanguage.java:1778)
> ... 1 more
> Caused by: java.lang.ClassNotFoundException: com.ibm.wsspi.webcontainer.facade.ServletContextFacade cannot be found by com.ibm.ws.webcontainer.servlet.4.0.jakarta_1.0.72.202212020909
> ... 20 more

--- a/dev/com.ibm.ws.webcontainer.servlet.4.0/bnd.bnd
+++ b/dev/com.ibm.ws.webcontainer.servlet.4.0/bnd.bnd
@@ -49,6 +49,7 @@ Import-Package: \
     javax.servlet.http; version="[2.6,3)", \
     javax.servlet.resources; version="[2.6,3)", \
     com.ibm.ws.webcontainer.resources, \
+    com.ibm.wsspi.webcontainer.facade, \
     *
 
 

